lib/constants.rb

```diff
@@ -0,0 +1,9 @@
+# frozen_string_literal: true
+
+module Constants
+  PAGE1 = 1
+  PAGE10 = 10
+  PAGE20 = 20
+  PAGE50 = 50
+  PAGE100 = 100
+end
```

Comment:
MAX_ITEMS_PER_PAGE = 10

and use it instead of PAGE10

and remove PAGE1...PAGE100 as it is overload